lib/color_mask.py

```diff
@@ -37,9 +42,14 @@ def get_frame(cap, scaling_factor):
         cv2.imshow('Original image', frame)
         cv2.imshow('Color Detector', res)
 
+
+
         # Check if the user pressed ESC key
         c = cv2.waitKey(5)
         if c == 27:
             break
 
     cv2.destroyAllWindows()
+
+if __name__=='__main__':
+    run_mask()
```

Comment:
One nice feature would be to accept command line arguments here, and pass them to the mask function to specify the magic number constants. That way, you could easily re-run the program until you found good values for the range ... at least, until we update this to calculate the range based on standard deviations.